rnbv.js

```diff
@@ -0,0 +1,178 @@
+#!/usr/bin/env node
```

Comment:
Given that we now have more scripts that live at `rnb-cli/src/tools` maybe we should move this one there